### ISSUE TITLE
HTML_TRACK_REMAINING resets selected tab every few seconds

### ISSUE THREAD

**Issue by phicoh** [freebsd/poudriere#625]
In the 'Remaining ports' section (feature HTML_TRACK_REMAINING) the selected tab gets reset every few seconds, making it hard to look beyond the first 10 remaining ports.

(Poudriere 3.2.7)

**Comment by bdrewery** [comment]
Yeah it's horribly inefficient and redraws it rather than updating it. (#386 related)
